src/widgets/RRule.tsx

```diff
@@ -0,0 +1,249 @@
+import * as React from 'react';
+import Container from './Container';
+import { TextField, Select, MenuItem, FormGroup, FormControlLabel, Checkbox } from '@material-ui/core';
+import RRule, { Options, Weekday } from 'rrule';
+
+import DateTimePicker from '../widgets/DateTimePicker';
+
+interface PropsType {
+  onChange: (rrule: string) => void;
+  rrule: string;
+}
+const frequency = {
```

Comment:
This type also exists.
https://github.com/jakubroztocil/rrule/blob/master/src/types.ts#L13

Reply:
```
const frequency = [ Frequency.YEARLY, Frequency.MONTHLY, Frequency.WEEKLY, Frequency.DAILY ];
```